### DISCUSSION TITLE
cpu占用率高

### DISCUSSION THREAD

**Discussion by jaesony** [lxhao61/integrated-examples#215] [Q&A] [answered]
用的Xray(M+F+B+G+A)+Caddy(N)，也只用了M+N。
vps最近经常报错：“Due to high CPU usage, server's CPU clock is currently being limited”
大概半个月前发现的，现在不得不一两天重启一次
查了下，是caddy的问题：
    PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND
   1658 caddy     20   0 1467548  75564  15096 S  99.0  15.6     15,56 caddy
请问大致是什么原因造成的，谢谢

**Comment by lxhao61** [comment]
不好判断。
1、你使用的是我最新示例？ 有 UDP 定向转发配置及 NaiveProxy 支持 HTTP/3 代理应用。
2、你使用的是我仓库的 Caddy 文件，还是自己编译的？版本多少及含那些插件等。

**Reply by jaesony** [reply]
1、是的，都是最新示例，有 UDP 定向转发配置及 NaiveProxy 支持 HTTP/3 代理应用
2、v2.8.4 h1:q3pe0wpBj1OcHFZ3n/1nl4V4bxBrYoSoab7rL9BMYNk=
我重新下载下caddy文件试试

**Reply by lxhao61** [reply, marked as answer]
1、使用今天最新修改示例试试。
2、可删除 UDP 定向转发配置及关闭 NaiveProxy 支持 HTTP/3 代理应用试试。
